Scripts/99 SL-ThemePrefs.lua

```diff
@@ -109,8 +109,13 @@ SL_CustomPrefs.Get = function()
 		{
 			Default = "",
 			Validation = function(val)
-				if SONGMAN:FindSong(val) then return true end
-				return false
+				-- We should probably use SONGMAN:FindSong() to
```

Comment:
I think that won't work because during startup when the function is called SONGMAN is nil and then the value gets rejected which results in `EditModeLastSeenSong` not being available in edit mode.